ext/opcache/jit/zend_jit_trace.c

```diff
@@ -5595,6 +5595,8 @@ static void zend_jit_trace_init_caches(void)
 	memset(JIT_G(bad_root_cache_count), 0, sizeof(JIT_G(bad_root_cache_count)));
 	memset(JIT_G(bad_root_cache_stop), 0, sizeof(JIT_G(bad_root_cache_count)));
 	JIT_G(bad_root_slot) = 0;
+	JIT_G(function_compilation_successes) = 0;
+	JIT_G(function_compilation_failures) = 0;
```

Comment:
This is not specific to tracing JIT. It would be better to move this assignments to zend_jit.c